code/src/Sword.ts

```diff
@@ -0,0 +1,18 @@
+import { Weapon } from './Weapon';
+
+export class Sword extends Weapon {
+  constructor(
+    baseDamage: number,
+    baseDurability: number,
+    value: number,
+    weight: number,
+  ) {
+    super('Sword', baseDamage, baseDurability, value, weight);
+  }
+
+  polish(): void {
+    let newDamage = this.getBaseDamage() + this.MODIFIER_CHANGE_RATE;
+    if (this.getBaseDamage() > 125) newDamage = 125;
```

Comment:
You need a slightly different condition here. You should calculate 25% of the base damage and check that damageModifier is not higher than this value.